package.json

```diff
@@ -36,7 +37,16 @@
     "uuid": "^8.0.0"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.21.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-react": "^7.20.0",
```

Comment:
@need4eat 
`babel` and `react` came as a part of the automated script I used. I'm conflicted between keeping it and removing it. Thoughts?